unity-renderer/Assets/DCLPlugins/ECS7/ProtocolBuffers/Definitions/Raycast.proto

```diff
@@ -0,0 +1,20 @@
+﻿syntax = "proto3";
+
+import "common/id.proto";
+option (ecs_component_id) = 1161;
+
+import "common/Vector3.proto";
+
+enum QueryType {
+    HIT_FIRST = 0;
+    QUERY_ALL = 1;
+}
+
+message PBRaycast {
+  int32 timestamp = 1; 
+  Vector3 origin = 2;
+  Vector3 direction = 3;
+  float max_distance = 4;
+  QueryType query_type = 5;
+
+}
```

Comment:
can we add a more specific name for that enum, like `RaycastQueryType` or puting it inside the message?
cause that would create a global enum named `QueryType` and we might want to have another non-raycast related type of queries